File: python/Stack.py
class Stack:
	def __init__(self):
		self.data = []

	def push(self, key):
		self.data.append(key)

	def pop(self):
		try:
			return self.data.pop()
		except:
			print("Stack underflow")

	def peep(self):
		try:
			return self.data[len(self.data) - 1]
		except:
			print("Stack underflow")

	def isempty(self):
		return self.data == []

	def __str__(self):
		return str(self.data)

if __name__ == '__main__':
	s = Stack()
	s.push(23)
	s.push(32)
	print(s)

File: java/stack.java
class stack
{
	public int[] data = new int[10];
	public int pos = -1;
	public void push(int key)
	{
		if(pos <= 9)
		{
			data[++pos] = key;
		}
		else
		{
			System.out.print("Stack is overflow");
		}
	}
	public int pop()
	{
		if(pos == -1)
		{
			System.out.print("Stack is underflow");
			return -1;
		}
		else
		{
			return data[pos--];
		}
	}
	public int peep()
	{
		if(pos < 0)
		{
			return data[pos];
		}
		else
		{
			System.out.print("Stack is underfow");
			return -1;
		}
	}

	public boolean isempty()
	{
		return pos == -1;
	}

	public boolean isfull()
	{
		return pos > 9;
	}

	public static void main(String[] args)
	{
		System.out.print("Create stack class and you can manuplate it");
	}
}
